platform/30-kyverno-setup.sh

```diff
@@ -55,5 +74,5 @@ kubectl patch \
   --type json --patch-file "${GIT_ROOT}"/platform/components/kyverno/patch_container_args.json
 
 echo -e "${C_GREEN}Creating verify-image admission control policy...${C_RESET_ALL}"
-cue export "$KYVERNO_RESOURCE_DIR"/admission-control-verify-image-resources.cue -e template -t key="$USERPUBKEY" --out yaml > "$KYVERNO_RESOURCE_DIR"/admission-control-verify-image-resources.yaml
-kubectl apply -f "$KYVERNO_RESOURCE_DIR"
+cue eval ./... -t repo="$REPO" -t key="$USERPUBKEY" --out yaml > $KYVERNO_RESOURCE_DIR/released/admission-control.yaml
```

Comment:
```suggestion
pushd "$KYVERNO_RESOURCE_DIR"
cue eval ./... -t repo="$REPO" -t key="$USERPUBKEY" --out yaml > $KYVERNO_RESOURCE_DIR/released/admission-control.yaml
popd
```

Reply:
Yup I needed to fix that. I was running all by hand.